src/v2/view-builder/views/ov/SelectEnrollmentChannelOktaVerifyView.js

```diff
@@ -2,24 +2,28 @@ import { loc, _ } from '@okta/courage';
 import { BaseForm } from '../../internals';
 import BaseAuthenticatorView from '../../components/BaseAuthenticatorView';
 import BrowserFeatures from '../../../../util/BrowserFeatures';
+import SameDeviceEnrollLink from './SameDeviceEnrollLink';
 
 const Body = BaseForm.extend({
   title() {
     return (BrowserFeatures.isAndroid() || BrowserFeatures.isIOS())
       ? loc('oie.enroll.okta_verify.setup.title', 'login')
-      : loc('oie.enroll.okta_verify.select.channel.title', 'login');
+      : loc('oie.enroll.okta_verify.select.channel.title.updated', 'login');
   },
   getUISchema() {
     const schemas = BaseForm.prototype.getUISchema.apply(this, arguments);
     // filter selected channel
     const channelField = _.find(schemas, (schema) => schema.name === 'authenticator.channel');
-    channelField.options = _.filter(channelField?.options, (option) =>
-      option.value !== this.options.appState.get('currentAuthenticator')?.contextualData?.selectedChannel);
+    // do not show the 'samedevice' radio option, that option is displayed as a link instead
+    channelField.options = _.filter(channelField?.options, (option) => (
+      option.value !== this.options.appState.get('currentAuthenticator')?.contextualData?.selectedChannel
+      && option.value !== 'samedevice'
+    ));
```

Comment:
I think my comment in `SameDeviceEnrollLink` should actually have been here
https://github.com/okta/okta-signin-widget/pull/3632/files#r1683443342

Reply:
I dont think it should be an issue because the user should not have access to the channel selection screen; when samedevice is the only channel, that means OV security is set to HIGH. The Or set up Okta Verify on a mobile device link to enroll in other channels will not display when security is set to HIGH